app/models/author.rb

```diff
@@ -1,4 +1,8 @@
 class Author < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable
```

Comment:
- [ ] The application need to ask for confirmation of email. So please add `:confirmable` to your devise as well